backend-public/contracts/community-handles.clar

```diff
@@ -34,31 +35,50 @@
         (try! (is-owner namespace))
         (try! (stx-transfer? u1 tx-sender (as-contract tx-sender)))
         (try! (as-contract (to-uint-response (contract-call? 'ST000000000000000000002AMW42H.bns name-preorder hash u1))))
-        (try! (as-contract (to-bool-response (contract-call?
-        'ST000000000000000000002AMW42H.bns namespace-update-function-price
-        namespace u0 u0 u0 u0 u0 u0 u0 u0 u0 u0 u0 u0 u0 u0 u0 u0 u0 u0 u1 u1))))
+        (try! (as-contract (to-bool-response (contract-call? 'ST000000000000000000002AMW42H.bns namespace-update-function-price namespace u0 u0 u0 u0 u0 u0 u0 u0 u0 u0 u0 u0 u0 u0 u0 u0 u0 u0 u1 u1))))
         (try! (as-contract (to-bool-response (contract-call? 'ST000000000000000000002AMW42H.bns name-register namespace name salt zonefile-hash))))
```

Comment:
salt can be removed as function parameter because the signature of .bns does not contain `owner` parameter

Reply:
Right, yeah, then let's do it. We can just hardcode a `0x00` constant. (Not even as a let var.)